modules/appeals_api/app/workers/appeals_api/higher_level_review_upload_status_updater.rb

```diff
@@ -23,7 +23,7 @@ def retry_limits_for_notification
       if Settings.vsp_environment == 'production'
         [2, 4]
       else
-        [3]
+        [5]
```

Comment:
IIRC the retry_count starts at 0 for the first retry, so w/ 5 retries this would be `4` and not `5`.

Reply:
we add 1 to the middleware when it comes in, so this would be the 5th retry, which would have a `retry_count` of 4